### DISCUSSION TITLE
Can connect to the unit when connected to pump but screen is blank

### DISCUSSION THREAD

**Discussion by scott-rosser** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#595] [Q&A]
I have setup everything, configured the wifi and when I connect it to the pump I can still connect via wifi but when I toggle the various buttons nothing happens and there is no screen on the actual pump unit. It is a 6 wire unit. I've used PCB V2 and no LLC. It's a pre-2021 model. Tried 2 boards now just in case. Any help would be greatly appreciated.

I've added an LLC now to see if that makes a difference and the board now beeps when I press the power button on the pump. I've added some pictures to see if I'm missing something? Also pump is model # 54154 if that helps.
![IMG_9636](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/142261131/7f1e1c21-22d3-48fd-ad8a-6e005bab463b)
![IMG_9637](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/142261131/e4b3596a-4f3f-475c-afef-a339366b520f)

**Comment by visualapproach** [comment]
That model is confirmed working with the red LLC. It doesn't mean it won't work with the blue. Just saying.
![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/22987250/b4dc83fd-044f-4546-97bc-2a308ef08106)

Does the switches in UI work? Does it show the right info? maybe just brightness set to zero?

**Reply by scott-rosser** [reply]
I did check that table but I didn't notice the red LLC bit. Just went for the blue as they are much quicker to get hold of, I'll order one and try it to rule it out though. From what I can see from the UI I don't think it's reading any information but I'm not sure if that's because the spa isn't actually running? I click on the switches in the UI and they just toggle back off after a few seconds. Brightness says it's set to 8. None of the buttons seem to do anything on the pump either it's just the board that makes the noise when you press the on/off button. 

<img width="786" alt="Screenshot 2023-08-25 at 19 38 38" src="https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/142261131/1c5a9410-0c24-45c1-874e-6acaf0386603">
